pom.xml

```diff
@@ -55,9 +55,36 @@
                 <suite.file>src/test/resources/AllTestsSuite.xml</suite.file>
             </properties>
         </profile>
-    </profiles>
 
+        <profile>
+            <id>HM2_exercise1</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.file>src/test/resources/HM2_exercise1.xml</suite.file>
```

Comment:
It will be better to store suite files in the proper folders. Ex `src/test/resouces/hw2`

Reply:
fix